static/app/components/profiling/flamegraph/aggregateFlamegraph.tsx

```diff
@@ -169,10 +172,25 @@ export function AggregateFlamegraph(props: AggregateFlamegraphProps): ReactEleme
       return null;
     }
 
-    return new FlamegraphRendererWebGL(flamegraphCanvasRef, flamegraph, flamegraphTheme, {
-      colorCoding,
-      draw_border: true,
-    });
+    const renderer = initializeFlamegraphRenderer(
+      [FlamegraphRendererWebGL, FlamegraphRenderer2D],
+      [
+        flamegraphCanvasRef,
+        flamegraph,
+        flamegraphTheme,
+        {
+          colorCoding,
+          draw_border: true,
+        },
+      ]
+    );
+
+    if (renderer === null) {
+      Sentry.captureException('Failed to initialize a flamegraph renderer');
```

Comment:
Can we potentially show an error message somewhere?

Reply:
I'll see if we can add an overlay to the flamegraph canvas, else I think a toast should be enough. We should never enter this case with the fallback renderers

Reply:
Im fine with a toast here 👍